src/APICore.ts

```diff
@@ -17,6 +21,13 @@ export class APICore {
 
     return backOff(req, {
       retry: (err: unknown) => err instanceof ThrottleError,
+      jitter: 'full',
+      timeMultiple: 1,
```

Comment:
Doesn't making the time multiple `1` make it less exponential? Wouldn't the default `2` work for this scenario?

Reply:
Yes, I suppose it does hehe

That being said, I believe having it retry on a fixed interval is the use-case we're trying to achieve 🤔 I could be mistaken though (CC @olamothe)

Reply:
After discussing with Olly, I'm going to make this option configurable 🙂 

Awaiting to hear back from the content ingestion team as to what they think the default value should be. But I'll update the default value based on their recommendation.